src/rpc/handlers/GatewayBalances.cpp

```diff
@@ -171,9 +171,9 @@ doGatewayBalances(Context const& context)
     if (!sums.empty())
     {
         boost::json::object obj;
-        for (auto const& [k, v] : sums)
+        for (auto const& [k, v1] : sums)
```

Comment:
instead of here, i feel we can make the original v more meaningful?
see line 35. 
`    auto v = ledgerInfoFromRequest(context);`